src/types/header.ts

```diff
@@ -0,0 +1,45 @@
+// Starknet
+import { BlockHeader } from "../deps.ts";
+
+// Eth
+import { bigIntToHex, Bloom, bytesToHex, JsonHeader } from "../deps.ts";
+
+const BLOCK_GAS_LIMIT = 30_000_000n;
+const BASE_FEE_PER_GAS = 100_000_000_000n;
+
+/**
+ * @param header - A Starknet block header.
+ * @param gasUsed - The total gas used in the block.
+ * @param logsBloom - The logs bloom of the block.
+ * @param receiptRoot - The transaction receipt trie root of the block.
+ * @param transactionRoot - The transaction trie root of the block.
+ * @returns The Ethereum block header in the json RPC format.
+ */
+export function toEthHeader(
+  { header, gasUsed, logsBloom, receiptRoot, transactionRoot }: {
+    header: BlockHeader;
+    gasUsed: bigint;
+    logsBloom: Bloom;
+    receiptRoot: Uint8Array;
+    transactionRoot: Uint8Array;
+  },
+): JsonHeader {
+  return {
+    parentHash: header.parentBlockHash,
+    uncleHash: "0x",
+    coinbase: header.sequencerAddress,
```

Comment:
I think this needs to be transformed into the EVM address of the sequencerAddress
to be sync'd with the team asap to establish once and for all (ssj+cairo0) the flow of the coinbase opcode